src/components/organisms/UiDatepicker/_internal/UiDatepickerCalendar.vue

```diff
@@ -231,19 +230,23 @@ function goToNextTab(): void {
   } else {
     dropdown.value?.closeHandler({ focusToggle: true });
   }
-}
-const inputsIds = inject('inputsIds') as ComputedRef<Record<string, string>>;
+};
+const inputsIds = inject<ComputedRef<Record<string, DatepickerDatePart>>>('inputsIds', computed(() => ({
+  'datepicker-input-day': 'day',
+  'datepicker-input-month': 'month',
+  'datepicker-input-year': 'year',
```

Comment:
Is this really a `Record<string, ...`  or maybe not a `string` but something more specific? Do we know it? Can it be specified with this code `Record<`datepicker-input-${DatepickerDatePart}`, DatepickerDatePart>`? Or am I nitpicking? 🙈 😂

Reply:
In this case, IMO I can't specify a more specific type than `string`. Actually, the default keys are the input IDs (`datepicker-input-${DatepickerDatePart}`), but you can override them by any `string` in the props.